build/package.json

```diff
@@ -85,5 +85,9 @@
     "tree-sitter-typescript": "^0.20.1",
     "vscode-gulp-watch": "^5.0.3"
   },
-  "dependencies": {}
+  "dependencies": {
+    "@actions/core": "1.2.6",
+    "@actions/github": "2.1.1",
+    "axios": "0.21.4"
+  }
```

Comment:
I ended up needing to install these because build folder wasn't compiling in my VM.

Reply:
I was able to resolve that by compiling actions folder first. But it should be fine to include these here too.